### DISCUSSION TITLE
Complie for other Linux distros

### DISCUSSION THREAD

**Discussion by MichalisDBA** [readysettech/readyset#1079] [Q&A]
Hello.

Is it possible to compile and create a binary for other Linux distros like RHEL, Centos, Almalinux? Basically RHEL and clones of it.

**Comment by aglover** [comment]
We build for Ubuntu, Centos, and Amazon Linux daily -- you should be able to produce RHEL RPMs via these directions: https://github.com/readysettech/readyset/blob/main/community-development.md -- if you run into trouble, jump onto our [community slack](https://join.slack.com/t/readysetcommunity/shared_invite/zt-2d4bxke4g-lOcXJsd7ztPTes8uCA60oA) and we should be able to sort you out!
